GameData/RealismOverhaul/RO_SuggestedMods/Squad/RO_Squad_Command.cfg

```diff
@@ -406,301 +283,92 @@
-
-	@MODULE[ModuleRCS*]
-	{
-		@name = ModuleRCSFX
-		%runningEffectName = running
-	}
-}
-
-@INTERNAL[PodCockpit|PodCockpitRPM]:FOR[RealismOverhaul]:NEEDS[VenStockRevamp,!ReStock]
-{
-	%scaleAll = 1.64, 1.64, 1.64
-	@MODULE[InternalSeat],*
-	{
-		%kerbalScale = 1.64, 1.64, 1.64
-		%kerbalOffset = 0.0, 0.0, 0.0
-		%kerbalEyeOffset = 0.0, 0.0, 0.0
-	}
-}
-
-@PART[landerCabinSmall]:FOR[RealismOverhaul]:NEEDS[!ReStock]
-{
-	%rescaleFactor = 1.6
-	@node_stack_bottom = 0.0, -0.625, 0.0, 0.0, -1.0, 0.0, 2
-	@node_stack_top = 0.0, 0.625, 0.0, 0.0, 1.0, 0.0, 2
-}
-
-@PART[landerCabinSmall]:FOR[RealismOverhaul]
-{
-	%RSSROConfig = True
-	!MODULE[TweakScale] {}
-
-	@mass = 1.2
-	%maxTemp = 800
-	%skinMaxTemp = 800
-
-	@manufacturer = Generic
-	@description = A small, single-person cabin for use on non-atmospheric landers and small spacecraft.
-	
-	!RESOURCE[ElectricCharge] {}
-	!MODULE[ModuleReactionWheel] {}
-	@MODULE[ModuleScienceContainer]
-	{
-		@storageRange = 2.08
-	}
-
-	@MODULE[ModuleCommand]
-	{
-		@minimumCrew = 0
-		RESOURCE
-		{
-			name = ElectricCharge
-			rate = 0.4
-		}
-	}
-
-	MODULE
-	{
-		name = ModuleFuelTanks
-		volume = 180
-		basemass = -1
-		type = ServiceModule
-		TANK
-		{
-			name = ElectricCharge
-			amount = 43200
-			maxAmount = 43200
-		}
-		TANK
-		{
-			name = Food
-			amount = 17.5499999996544
-			maxAmount = 17.5499999996544
-		}
-		TANK
-		{
-			name = Water
-			amount = 11.5998000003072
-			maxAmount = 11.5998000003072
-		}
-		TANK
-		{
-			name = Oxygen
-			amount = 1776.59574468077
-			maxAmount = 1776.59574468077
-		}
-		TANK
-		{
-			name = CarbonDioxide
-			amount = 0
-			maxAmount = 286.458226550415
-		}
-		TANK
-		{
-			name = Waste
-			amount = 0
-			maxAmount = 1.5960000001536
-		}
-		TANK
-		{
-			name = WasteWater
-			amount = 0
-			maxAmount = 14.771999999808
-		}
-		TANK
-		{
-			name = LithiumHydroxide
-			amount = 2.25
-			maxAmount = 2.25
-		}
-	}
-
-	%setDefaultTACLS = true
-	%defaultTACLSPax = 1.0
-}
-
-+PART[landerCabinSmall]:FOR[RealismOverhaul]
-{
-	@name = landerCabinMedium
-}
-
-@PART[landerCabinMedium]:AFTER[RealismOverhaul]
-{
-	%RSSROConfig = true
-	%CrewCapacity = 2
-	%title = Mk1 Lander Can Advanced
-	%manufacturer = Generic
-	%description = This capsule was designed for lightweight non-atmospheric landers, and seats two occupants.
-
-	%mass = 1.6
-	%maxTemp = 800
-	%skinMaxTemp = 800
-
-	@MODULE[ModuleCommand]
-	{
-		@RESOURCE[ElectricCharge]
-		{
-			%rate = 0.7
-		}
-	}
-	!INTERNAL[landerCabinSmallInternal] {}
-	INTERNAL
-	{
-		name = landerCabinMediumInternal
-	}
-
-	@MODULE[ModuleFuelTanks]
-	{
-		@volume = 1200
-		@TANK:HAS[~name[ElectricCharge]],*		// Double everything except EC
-		{
-			@amount *= 2
-			@maxAmount *= 2
-		}
-	}
-	@MODULE[TacGenericConverter]:NEEDS[TacLifeSupport]
-	{
-		@conversionRate = 2.0
-	}
-	!MODULE[ModuleConnectedLivingSpace] {}
-	MODULE:NEEDS[ConnectedLivingSpace]
+	@INTERNAL[landerCabinSmallInternal] { @name = landerCabinMediumInternal }
```

Comment:
per #2323, this won't work if the RPM variant is used (landerCabinSmallInternalRPM)